docs/migrations/migrations.md

```diff
@@ -0,0 +1,232 @@
+---
+myst:
+  html_meta:
+    "description": "Migrating Plone"
+    "property=og:description": "Migrating Plone"
+    "property=og:title": "Migrating Plone"
+    "keywords": "Versions, Upgrade, Migration, Relaunch, Changes"
+---
+
+(migrations-label)=
+
+# Migrating Plone
+
+## Intro
+
+* What do you want to learn?
+* What are your experiences with migrations so far?
+* What are your next use cases?
+
+
+## What are Migrations?
+
+### Plone Versions
+
+Like most software, Plone changes over time.
+These changes are reflected in different releases of Plone.
+
+The [release-schedule](https://plone.org/download/release-schedule) specifies, which versions are supported and for how long.
+
+```{figure} https://plone.org/download/plonereleaseschedule-2022-09-12.png/@@images/image
+:alt: The Plone release schedule
+```
+
+Maintenance support
+: Bug fixes and small new features are added.
+  Around the end of maintenance support, a final release will be done.
+
+Security support
+: Plone security hotfixes will be made available for this series.
+  For more information, see the [security update policy](https://plone.org/security/update-policy).
+
+Currently, Plone 6.0.x is under maintenance support and Plone 5.2.x is under security support.
```

Comment:
The graphic with timeline just above this text shows that Plone 5.2 will be in maintenance support until december 2023. That's different from this text

Reply:
I will change it to "From the time Plone 6 is released..."